viz.yaml

```diff
@@ -334,18 +334,9 @@ process:
     depends: sp-sites
     processor: order_sites
   - 
-    id: gage-sparks
-    location: cache/gage-sparks.rds
-    processor: gage_sparks
-    scripts: scripts/process/process.sparks.R
-    reader: rds
-    depends: 
-      gage-data: storm-gage-height
-      sites: order-sites
-  - 
-    id: flood-sparks
-    location: cache/flood-sparks.rds
-    processor: flood_sparks
+    id: sparks
```

Comment:
why aren't we displaying flood vs non flood anymore?

Reply:
We are, look down in the hurricane map visualize. I split it apart later.